books/free-programming-books-subjects.md

```diff
@@ -14,6 +14,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Cloud Computing](#cloud-computing)
 * [Competitive Programming](#competitive-programming)
 * [Compiler Design](#compiler-design)
+* [Compiler Graphics](#compiler-graphics)
```

Comment:
1. please check your work
2. Why not use "graphics programming'?

Reply:
1. Hi. I'm sorry but I don't understand. Check my work as in?
2. Graphics Programming is a domain where people "code" or create things graphically. Like Scratch kind of is. Another example will be a no-code game engine. JPEG and all its concepts are about how to efficiently store, display and transmit visual information in form of images. the domain "Computer Graphics" suits it a lot more.

Here's an article showing what Graphics Programming is: https://www.finsliqblog.com/programming-languages/what-is-graphics-programming/#:~:text=Graphic%20or%20visual%20programming%20is,arrangement%20of%20the%20visual%20objects.

Thanks (I am new to Contributing to Open Source so I may make many mistakes haha please bear with me)

Reply:
because we already have a "graphics programming" section.